packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -48,6 +51,28 @@ typedef SetTextHandler = void Function(String text);
 /// Returned by [SemanticsConfiguration.getActionHandler].
 typedef SemanticsActionHandler = void Function(Object? args);
 
+/// The semantics merge rule generated from [SemanticsConfiguration.merger].
+class SemanticsMergeRule {
```

Comment:
I would have expected a constructor here where you have to pass in the two lists. I don't think we should encourage people to manipulate the lists directly.